base/broadcast.jl

```diff
@@ -172,7 +172,7 @@ struct Broadcasted{Style<:Union{Nothing,BroadcastStyle}, Axes, F, Args<:Tuple} <
     args::Args
     axes::Axes          # the axes of the resulting object (may be bigger than implied by `args` if this is nested inside a larger `Broadcasted`)
 
-    Broadcasted(style::Union{Nothing,BroadcastStyle}, f::Tuple, args::Tuple) = error() # disambiguation: tuple is not callable
+    Broadcasted(style::Union{Nothing,BroadcastStyle}, f::Tuple, args::Tuple) = throw(ArgumentError("invalid types")) # disambiguation: tuple is not callable
```

Comment:
Use the comment text as the message:

```suggestion
    Broadcasted(style::Union{Nothing,BroadcastStyle}, f::Tuple, args::Tuple) = throw(ArgumentError("invalid types; tuple is not callable"))
```